packages/tools/client-debugger/client-debugger-view/src/components/ContainerView.tsx

```diff
@@ -139,53 +140,26 @@ export interface PanelViewSelectionMenuProps {
  * @internal
  */
 export function PanelViewSelectionMenu(props: PanelViewSelectionMenuProps): React.ReactElement {
-	const { currentSelection, updateSelection } = props;
+	const { updateSelection } = props;
+	const [selectedValue, setSelectedValue] = React.useState<TabValue>("none");
 
+	const onTabSelect = (event: SelectTabEvent, data: SelectTabData): void => {
+		setSelectedValue(data.value);
+		updateSelection(data.value as PanelView);
+	};
 	const options: IOverflowSetItemProps[] = Object.entries(PanelView).map(([_, flag]) => ({
 		key: flag,
 	}));
```

Comment:
These `options: IOverflowSetItemProps` were necessary with the previous approach of using an `OverflowSet` component. Can we simplify so we don't need the type anymore, since we moved to a different component?